config/install_debug.yaml

```diff
@@ -12361,7 +12361,8 @@ metadata:
   name: kyverno:events
 rules:
 - apiGroups:
-  - '*'
```

Comment:
Do we need to change this in this PR ?

Reply:
sorry...not sure why that's happening

Reply:
Can you remove those changes from this PR ?

Reply:
👍